libs/brig-types/src/Brig/Types/User.hs

```diff
@@ -154,6 +159,7 @@ instance ToJSON User where
         # "locale"    .= userLocale u
         # "service"   .= userService u
         # "handle"    .= userHandle u
+        # "expire"    .= (UTCTimeMillis <$> userExpire u)
```

Comment:
Could we call this `expires_at` instead?

Reply:
We certainly can.